src/index.html

```diff
@@ -20,8 +20,389 @@
+          </div>
+          <div class="card__hover">
+            <a
+              class="card__button card__button--margin-top"
+              href="#"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+        <!-- 7 -->
+        <div class="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Product Image"
+          />
+          <div class="card__description card__description--margin-top">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code card__code--margin-top">
+            Product code: 195434
+          </div>
+          <div class="card__rating card__rating--margin-top">
+            <ul class="stars">
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+          <div class="card__price-block card__price-block--margin-top">
+            <div class="card__txt">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <div class="card__hover">
+            <a
+              class="card__button card__button--margin-top"
+              href="#"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+        <!-- 8 -->
+        <div class="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Product Image"
+          />
+          <div class="card__description card__description--margin-top">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code card__code--margin-top">
+            Product code: 195434
+          </div>
+          <div class="card__rating card__rating--margin-top">
+            <ul class="stars">
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star stars__star--active"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+          <div class="card__price-block card__price-block--margin-top">
+            <div class="card__txt">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <div class="card__hover">
+            <a
+              class="card__button card__button--margin-top"
+              href="#"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
```

Comment:
All product cards have identical content. If this is intentional for testing purposes, it's fine. However, if this is meant to represent different products, consider using a templating system or JavaScript to dynamically generate these cards with unique content.